com.unity.render-pipelines.high-definition/Editor/RenderPipeline/PathTracing/PathTracingEditor.cs

```diff
@@ -57,6 +66,35 @@ public override void OnInspectorGUI()
                         PropertyField(m_MaxDepth);
                         PropertyField(m_MaxIntensity);
                         PropertyField(m_SkyImportanceSampling);
+#if ENABLE_UNITY_DENOISING_PLUGIN
+                        PropertyField(m_Denoising);
+                        var denoiserType = m_Denoising.value.GetEnumValue<DenoiserType>();
+                        bool supported = Denoiser.IsDenoiserTypeSupported(denoiserType);
+
+                        if (m_Denoising.value.intValue != (int) DenoiserType.None)
+                        {
+                            using (new IndentLevelScope())
+                            {
+                                if (supported)
+                                {
+                                    PropertyField(m_UseAOV);
+                                    if (m_Denoising.value.intValue == (int)DenoiserType.Optix)
+                                    {
+                                        PropertyField(m_Temporal);
+                                    }
+                                }
+                                else
+                                {
+                                    EditorGUILayout.HelpBox($"The selected denoiser is not supported by this hardware configuration.", MessageType.Error, wide: true);
+                                }
+                            }
+                        }
+#else
+                        CoreEditorUtils.DrawFixMeBox("Denoising is not active in the path tracer. To activate it, install the Unity Denoising Plugin package.", MessageType.Info, () =>
```

Comment:
Suggestion for the message (to reuse the exact same formulation as with DLSS):
"Path Tracing Denoising is not active in this project. To activate it, install the Unity Denoising Plugin package."